src/targets/awsLambdaLayer.ts

```diff
@@ -7,7 +7,7 @@ import {
   getAuthUsername,
   getGithubApiToken,
   getGithubClient,
-  GithubRemote,
+  GitHubRemote,
```

Comment:
I was just annoyed by the incorrect capitalization but now I realize there is more to fix. I'll revert this change.